.github/workflows/firebase-hosting-merge.yml

```diff
@@ -0,0 +1,41 @@
+# This file was auto-generated by the Firebase CLI
+# https://github.com/firebase/firebase-tools
+
+name: Deploy to Firebase Hosting on merge
+
+on:
+  push:
+    branches: ["master", "release/**"]
+
+defaults:
+  run:
+    working-directory: ./my-webxr-app
+
+jobs:
+  build_and_deploy_to_staging:
+    if: startsWith(github.base_ref, 'release/')
```

Comment:
Are the `if`s needed if we have the `on` rules earlier?

Reply:
the `on` rule was on both branches (master and release). I have to add this condition otherwise, a merge into 'master' is also going to trigger a deployment in Staging environment